runtime/src/bank.rs

```diff
@@ -5287,6 +5287,12 @@ impl Bank {
             .add_program(program_id, process_instruction_with_context);
     }
 
+    /// Replace a builtin instruction processor if it already exists
+    pub fn remove_builtin(&mut self, name: &str, program_id: Pubkey) {
+        debug!("Removing program {} under {:?}", name, program_id);
```

Comment:
i was thinking whether to remove or not `KeccakSecp256k11111111111111111111111111111` from AccountsDB here

- what about `Ed25519SigVerify111111111111111111111111111`? (upcoming precompiled program) Should it exist in AccountsDB or not?  (consistency with currently existing `KeccakSecp256k11111111111111111111111111111`)
- maybe, it's good idea for all program_ids in a tx to be `executable` for consistent parsing/displaying (i.e. explorer)/inspecting. So, all precompile programs should exist in AccountsDB?

so, my verdict is to leave it there? then, how about explicitly commenting why we're NOT removing from AccountsDB?

Reply:
Recent changes will fail any tx that contains a program who's account is not executable, so I think we need to add accounts as executable for all supported precompiles

Reply:
so, commenting seems to the right path. :)

Reply:
could you comment about the exact reason we don't remove accounts from accountsdb?

Reply:
No reason in this case, I was originally thinking that the precompile would be added and then the builtin of the same name would be removed.  I've added back the remove of the builtin account.